Documentation/running-kvm-stage1.md

```diff
@@ -78,3 +78,22 @@ This will build the rkt binary and the KVM stage1 aci image in `build-rkt-1.25.0
 [rkt-arch-stage1]: devel/architecture.md#stage-1
 [rkt-run]: subcommands/run.md#use-a-custom-stage-1
 [stage1-implementers-guide]: devel/stage1-implementors-guide.md
+
+## Additional parameters
+
+The KVM stage1 has some hypervisor specific parameters that can change the execution environment.
+
+### Extra kernel command line parameters
+
+Additional Linux kernel command line parameters (`/proc/cmdline`) can be passed via the environment variable `RKT_HYPERVISOR_EXTRA_KERNEL_PARAMS`:
```

Comment:
Hi @alban . Rather that '/proc/cmdline', I might have substituted the kernel documentation link to kernel-parameters.txt (or possibly more upto date the .rst file in that place) instead - but, it's not a blocker for me, just a suggestion.